tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "lib": ["dom", "dom.iterable", "esnext"],
     "allowJs": true,
     "skipLibCheck": true,
-    "strict": true,
+    "strict": false,
```

Comment:
question(blocking): Why do we need to disable strict mode for typechecking? I'd prefer keeping this = `true`

Reply:
It's set to false for both the market and the portal repo, if we set it to true it brings up more errors, we can do it but I'm not sure I can fix them all.

Reply:
Okay, keep on false for now. Will add a low prio issue tracking this